tests/runtests.py

```diff
@@ -10,7 +10,12 @@
 import tempfile
 import warnings
 
-import django
+try:
+    import django
+except ImportError:
+    sys.stdout.write("Django module not found, please reference tests/README.rst for instructions.\n")
```

Comment:
is there a reason to use `sys.stdout.write` instead of just print? also error messages should probably go to `stderr`

Reply:
No, and I agree `stderr` would be better. Will change that.

Reply:
I think `print(… file=sys.stderr)` would be better.

Also what about something like:
```
 try:
    import django
except ImportError as e:
   raise RuntimeError("Django module not found, please reference tests/README.rst for instructions.") from e
```

which would keep the traceback?

Reply:
That would probably be the desired behavior.